Blockzilla/OverlayView.swift

```diff
@@ -257,6 +259,15 @@ class OverlayView: UIView {
 
     @objc private func didPressSearch(sender: IndexedInsetButton) {
         delegate?.overlayView(self, didSearchForQuery: searchSuggestions[sender.getIndex()])
+        
+        if(sender.getIndex() == 0) {
+            //Ping that default text has been selected instead of search suggestion
```

Comment:
These comments likely are not necessary since the Telemetry call itself explains what is happening.